src/rpc/net.cpp

```diff
@@ -100,6 +100,8 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
                             {RPCResult::Type::BOOL, "relaytxes", "Whether peer has asked us to relay transactions to it"},
                             {RPCResult::Type::NUM_TIME, "lastsend", "The " + UNIX_EPOCH_TIME + " of the last send"},
                             {RPCResult::Type::NUM_TIME, "lastrecv", "The " + UNIX_EPOCH_TIME + " of the last receive"},
+                            {RPCResult::Type::NUM_TIME, "last_transaction", "The " + UNIX_EPOCH_TIME + " of the last transaction accepted and relayed by this peer"},
+                            {RPCResult::Type::NUM_TIME, "last_block", "The " + UNIX_EPOCH_TIME + " of the last block accepted and relayed by this peer "},
```

Comment:
I don't think 'accepted and relayed by' is quite right:

For `last_transaction`, it's the time of the last transaction that we received and accepted into our mempool. I think `the time of the last valid transaction received from this peer` is accurate.
For `last_block`, it's the time of the last block that we received and saved to disk (even if we don't connect the block or it eventually fails connection). I think `the time of the last block received from this peer` is fine here.

Reply:
Thanks @jnewbery! I was just reconsidering those as well. Updating.

Reply:
Unresolving to make the head comment visible for reviewers and future code doc writers.